crates/store/re_video/src/demux/mod.rs

```diff
@@ -96,8 +90,40 @@ impl VideoData {
 
     /// The codec used to encode the video.
     #[inline]
-    pub fn codec(&self) -> &str {
-        &self.config.codec
+    pub fn human_readable_codec_string(&self) -> String {
```

Comment:
I'd actually would have gone further and turn it around: e.g. `AV1 (av01.123.stuff)`